templates/zerver/help/change-the-privacy-of-a-stream.md

```diff
@@ -2,51 +2,26 @@
 
 {!admin-only.md!}
 
-Follow the following steps to change who can join your stream by changing the stream's
-privacy.
+By default, a stream's privacy level is set at creation time by the
+stream creator.
 
-Please note that any users that were previously subscribed to the stream will
-remain subscribed to the stream after you change the stream's privacy.
+An organization administrator can also change the privacy level of a stream
+under certain conditions. The organization administrator can always make a
+public stream private. However, the organization administrator must be a
+subscriber to a private stream in order to take that private stream public or
+convert it into a different type of private stream. If a private stream needs
```

Comment:
This doesn't have it's own search term any more, but I thought that it was smoother to mention the permissions model in the beginning, rather than having separate sections for each possible privacy model change. With three privacy models, that's a total of 6 changes.